repos/system_upgrade/common/actors/initramfs/upgradeinitramfsgenerator/libraries/upgradeinitramfsgenerator.py

```diff
@@ -1,3 +1,6 @@
+from collections import namedtuple
+import contextlib
+import itertools
 import os
 import shutil
 from distutils.version import LooseVersion
```

Comment:
Just a note; since this code has not been changed, we don't need to fix it here:

I noticed that pylint complains that distutils.version is deprecated.  If we require the rpm python bindings (I believe yum and dnf already do) then we can use `rpm.labelCompare` with `functools.cmp_to_key` instead of `LooseVersion` to do this sorting.

Reply:
this is actually resolved in the master already. rebase shuld fix it. See commit 236483d5478ce845726ee05b7ca3e080105cdb80